### ISSUE TITLE
dom.indexedDB.enabled

### ISSUE THREAD

**Issue by The-OP** [The-OP/Fox#124]
@leedoyle commented on 74ff7e1 Jun 23, 2017
>Как ни странно, mega работает в TBB, несмотря на dom.indexedDB.enabled

Да, это достаточно интересно. Вот что я обнаружил: если в обычном Firefox при отключенной `dom.indexedDB.enabled` обратиться к `window.indexedDB`, вылетит исключение. В TBB же при таком обращении просто возвращается `null` (видимо вследствие каких-то их патчей).

Если сделать так:
```
// ==UserScript==
// @include https://mega.nz/*
// @run-at document-start
// @grant none
// ==/UserScript==
Object.defineProperty(window, 'indexedDB', {value: null});
```

то Mega начинает работать и с выключенным `dom.indexedDB.enabled`.

**Comment by leedoyle** [comment]
Да, очень интересно. Возможно с таким патчем заработают и Twitter и Google Drive?

**Comment by The-OP** [comment]
Действительно работают. По крайней мере скачивание файлов в Google Drive и прокрутка таймлайна в Twitter. Остальное не тестировал.

**Comment by The-OP** [comment]
Пускай скрипт тут полежит: 82e2959
Может пригодится.
